src/widgets/response/response.tsx

```diff
@@ -1,6 +1,7 @@
 import type { RestResponse } from '~/app/rest/actions';
 import { CodeEditor } from '../codeEditor/editor';
 
+import { HeadersViewer } from '../headers/headers';
```

Comment:
```suggestion
import { CodeEditor } from '~/widgets/codeEditor/editor';

import { HeadersViewer } from '~/widgets/headers/headers';
```
По FSD использование относительных импортов разрешено только в рамках самого виджета но не для импортов внешних компонент.